cfgov/unprocessed/js/modules/VideoPlayer.js

```diff
@@ -0,0 +1,206 @@
+/* ==========================================================================
+    Base Video Player Class
+   ========================================================================== */
+
+'use strict';
+
+var _assign = require( './util/assign' ).assign;
+var _noopFunct = require( './util/standard-type' ).noopFunct;
+var DOM_INVALID = require( '../config/error-messages-config' ).DOM.INVALID;
+var jsLoader = require( './util/js-loader' );
+
+var CLASSES = Object.freeze( {
+    VIDEO_PLAYER_SELECTOR:     '.video-player',
+    VIDEO_PLAYING_STATE:       'video-playing',
+    IFRAME_CLASS_NAME:         'video-player_iframe',
+    IFRAME_CONTAINER_SELECTOR: '.video-player_iframe-container',
+    PLAY_BTN_SELECTOR:         '.video-player_play-btn',
+    CLOSE_BTN_SELECTOR:        '.video-player_close-btn'
+} );
+
+function VideoPlayer( element, options ) {
+  _this = this;
+  options = options || {};
+  this.baseElement = _ensureElement( element, options.createIFrame );
+
+  // TODO: Add utility function for dataset. Oh dear IE, how I love you.
+  this.iFrameProperties = _assign( this.baseElement.dataset || {}, this.iFrameProperties );
+  _setChildElements( this.childElements );
+  _initEvents();
+  this.init.apply( this, arguments );
+}
+
+VideoPlayer.extend = function extend( attributes ) {
+  function child() {
+    this._super = VideoPlayer.prototype;
+    return VideoPlayer.apply( this, arguments );
+  }
+  child.prototype = Object.create( VideoPlayer.prototype );
+  _assign( child.prototype, attributes );
+  child.init = VideoPlayer.init;
+
+  return child;
+};
+
+VideoPlayer.init = function init( selector ) {
+  selector = selector || '[class*="video-player"]:not([class*="video-player__"])';
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/36f6195becc4c28009ba7b84c31290ffe07277ae